### PR TITLE
Fix/Remove hover bg and name color for locked option

### PR DESCRIPTION
**Pull request by danicunhac**
A locked option should not have a background color on hover, neither it's name should change the color, only the upgrade/start trial CTA.

## Preview
Screen Shot 2022-06-07 at 12 54 56

## Related PR's
https://github.com/AdaloHQ/frontend/pull/1260

### CODE REVIEW

**Comment by danicunhac** [comment]
[[ENG] Add trial and feature access to location property](https://trello.com/c/dazSQ1D9/447-eng-add-trial-and-feature-access-to-location-property)

File: src/styles/styles.css [hunk 1]
```diff
@@ -85,11 +85,15 @@
   color: #a82058;
 }
 
-.multi-menu-wrapper.red .multi-menu-item.menu-option:hover {
+.multi-menu-wrapper.red .multi-menu-item:not(.locked).menu-option:hover {
   background-color: rgba(239, 76, 48, 0.15);
   color: #ef4c30;
 }
```
Comment (tobyjsullivan) [inline src/styles/styles.css:91]: What do you think about using additive rules for this? Maybe its just personal preference but I find them simpler to reason about.

```suggestion
.multi-menu-wrapper.red .multi-menu-item.menu-option:hover {
  background-color: rgba(239, 76, 48, 0.15);
  color: #ef4c30;
}

.multi-menu-wrapper.red .multi-menu-item.locked.menu-option:hover {
  background-color: transparent;
}
```

Comment (danicunhac) [reply src/styles/styles.css:91]: I think these locked rules are more general, it'd be redundant to add the transparent background to all multi menu's themes

**Review by tobyjsullivan — APPROVED** [review]
LGTM 👍

**Pull request merged**